puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -213,6 +233,21 @@
         multiple_backends      => $multiple_backends,
         iscsi_bind_addr        => map_params('local_bind_addr'),
         glusterfs_shares       => $glusterfs_shares,
+        netapp_hostname           => $netapp_hostname,
+        netapp_login              => $netapp_login,
+        netapp_password           => $netapp_password,
+        netapp_server_port        => $netapp_server_port,
+        netapp_storage_family     => $netapp_storage_family,
+        netapp_transport_type     => $netapp_transport_type,
+        netapp_storage_protocol   => $netapp_storage_protocol,
+        netapp_nfs_shares         => $netapp_nfs_shares,
+        netapp_nfs_shares_config  => $netapp_nfs_shares_config,
+        netapp_volume_list        => $netapp_volume_list,
+        netapp_vfiler             => $netapp_vfiler,
+        netapp_vserver            => $netapp_vserver,
+        netapp_controller_ips     => $netapp_controller_ips,
+        netapp_sa_password        => $netapp_sa_password,
+        netapp_storage_pools      => $netapp_storage_pools,
```

Comment:
These are the ones I was looking for to be added here, this looks better. I'll do another thorough read-through and test (to make sure this doesn't break non-netapp setup, as I don't have netapp to test with), then hopefully we can get this merged tomorrow.  Thanks for making the updates, I'll comment if I find any other issues.